packages/firefox.rb

```diff
@@ -3,16 +3,22 @@
 class Firefox < Package
   description 'Mozilla Firefox (or simply Firefox) is a free and open-source web browser'
   homepage 'https://www.mozilla.org/en-US/firefox/'
-  version '87.0'
+  version '87.0' if ARCH =~ /^(i686|x86_64)/
   license 'MPL-2.0, GPL-2 and LGPL-2.1'
-  compatibility 'i686,x86_64'
+  compatibility 'all'
+
   case ARCH
   when 'i686'
     source_url "https://archive.mozilla.org/pub/firefox/releases/#{version}/linux-i686/en-US/firefox-#{version}.tar.bz2"
     source_sha256 '9127aee106dd9f09fac0c3cb89c5d75553384da4ec9be5943b60a5f55f31fccc'
   when 'x86_64'
     source_url "https://archive.mozilla.org/pub/firefox/releases/#{version}/linux-x86_64/en-US/firefox-#{version}.tar.bz2"
     source_sha256 '3c9207bee0a998634c4fd12293acfae207d16508749ad405bf1e8717d06acf02'
+  when 'armv7l', 'aarch64'
+    @_arm = true
+    version '87.0-2' # follow debian patch
+    source_url "http://http.us.debian.org/debian/pool/main/f/firefox/firefox_#{version}_armhf.deb"
```

Comment:
If we're going to start using Debian packages, we should be consistent and use them for i686 and x86_64 also.  It would be a maintenance nightmare to manage, otherwise.

Reply:
Moved to `deb`